app/js/controllers/EditCtrl.js

```diff
@@ -27,6 +28,24 @@ KMCMenu.controller('EditCtrl', ['$scope','$http', '$timeout','PlayerData','Playe
 			var playerName = versionsObj[PlayerService.KALTURA_PLAYER] ? PlayerService.KALTURA_PLAYER : PlayerService.KALTURA_PLAYER_OTT;
 			var playerVersion = versionsObj[playerName] === "{beta}" ? 'beta' : 'latest';
 			var autoUpdate = (versionsObj[playerName] === "{beta}" || versionsObj[playerName] === "{latest}");
+			PlayerService.getTagToProductVersion().success(function (tagToProductVersion) {
+				if (!autoUpdate){
+					try {
+						if ($scope.playerData.html5Url){
+							$scope.playerData.html5Url = JSON.parse($scope.playerData.html5Url);
+							if ($scope.playerData.html5Url.version) {
+								$scope.playerData['frozenProductVersion'] = $scope.playerData.html5Url.version;
+							}
+						}
+						$scope.playerData['frozenProductVersion'] = $scope.playerData['frozenProductVersion'] || tagToProductVersion[versionsObj[playerName]] || UNKNOWN_PRODUCT_VERSION;
```

Comment:
move this line after the try-catch and leave the catch empty

Reply:
done